test/e2e/storage/vsphere_stress.go

```diff
@@ -0,0 +1,180 @@
+package storage
+
+import (
+	"fmt"
+	. "github.com/onsi/ginkgo"
+	. "github.com/onsi/gomega"
+	"k8s.io/api/core/v1"
+	storageV1 "k8s.io/api/storage/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/types"
+	k8stype "k8s.io/apimachinery/pkg/types"
+	clientset "k8s.io/client-go/kubernetes"
+	"k8s.io/kubernetes/pkg/cloudprovider/providers/vsphere"
+	"k8s.io/kubernetes/test/e2e/framework"
+	"os"
+	"strconv"
+	"sync"
+)
+
+/*
+	Induce stress to create volumes in parallel with multiple threads based on user configurable values for number of threads and iterations per thread.
+	The following actions will be performed as part of this test.
+
+	1. Create Storage Classes of 4 Categories (Default, SC with Non Default Datastore, SC with SPBM Policy, SC with VSAN Storage Capalibilies.)
+    2. READ VCP_STRESS_INSTANCES and VCP_STRESS_ITERATIONS from System Environment.
```

Comment:
nit: indentation.

Reply:
It might be useful to list all of the env variables these tests require:
. VCP_STRESS_INSTANCES
. VCP_STRESS_ITERATIONS
. VSPHERE_SPBM_POLICY_NAME
. VSPHERE_DATASTORE

Reply:
Fixed indentation and added all env variables test requires in the comment